### ISSUE TITLE
Does not mock req.accepts()

### ISSUE THREAD

**Issue by j-quelly** [davesag/mock-req-res#83]
Does not mock req.accepts()

**Comment by davesag** [comment]
@j-quelly thanks for this.  In the short term you can deal with this yourself by

```js
const { expect } = require('chai')
const { spy } = require('sinon')
const { mockRequest } = require('mock-req-res')

const req = mockRequest({ accepts: spy() })

//....
```

I'll add `accepts` over the weekend and release an update.

**Comment by davesag** [comment]
@j-quelly fixed in version 1.2.0. Please update and, again, thanks for the suggestion.
